tests/nn/data_parallel/test_fsdp.py

```diff
@@ -697,6 +697,38 @@ def _test_no_sync(self, model, batch_dim):
         assert objects_are_equal(ref_grads, accumulated_grads, raise_exception=True)
 
 
+class TestAutoWrap(unittest.TestCase):
```

Comment:
if this test is not dependent on the rest of the file, move it to test_fsdp_config_context.py? This file is getting too big.